content/contribute/sponsoring.md

```diff
@@ -11,7 +11,7 @@ layout: "general"
 ### Why sponsor?
 
 *The GRASS GIS project is a volunteer-based organization*, 
-it is an affiliated project and founding member of the
+it is an affiliated project and a project of the
```

Comment:
```suggestion
it is a project of the
```
just to simplify...

Reply:
That was definitively the intention just like in the other place.